### DISCUSSION TITLE
基于 Dataview 构建 Obsidian Memos 的监控大盘

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#18] [General]
# 基于 Dataview 构建 Obsidian Memos 的监控大盘

obsidian memos 是我的日常 capture、fleeting idea 集散地，总结回顾以来 memos 的状态，这就需要一个好的监控大盘。这篇文章就是记录如何利用 dataview sql-like 的可编程能力搭建自己的 memos 监控大盘的。

https://www.edony.ink/my-obsidian-memos-monitor-system/

**Comment by 1614080902102** [comment]
写的很赞，学到了，今晚就去实践，谢谢大佬👍
